src/com/projectkorra/projectkorra/firebending/FireManipulation.java

```diff
@@ -179,6 +186,11 @@ public void progress() {
 			}
 		}
 	}
+	
+	@Override
+	public boolean isEnabled() {
+		return ConfigManager.getConfig().getBoolean("Abilities.Fire.FireManipulation.Enabled");
```

Comment:
Why are you overriding this?

Reply:
Should I not? It's the only way I've used it and I researched other code and they seemed to have overrided code from the same class I pulled that. Unless i'm crazy.

Reply:
With your latest commit, you got the wrong idea. He means "why are you overriding the default method?", as in, why is it there in the first place. It returns the same thing as the default method, so there's no need for it, is there?

Reply:
Well now I'm not sure, confused. What I know is: Before there wasn't an enable option and now there is. It worked for me just fine so idk what other ways to do it there are.

Reply:
So there wasn't an enabled option in the config beforehand that _worked_?

Reply:
From what I can see, there wasn't an enabled option in the config beforehand (which you added). However, now that there is one, it will be used without you having to specify an `isEnabled` method (which is why it is not needed). If you remove the `isEnabled` method, it _should_ still work

Reply:
Yes, there was no enable config option I wasn't aware that I didn't have to include it in the code, thank you.

Reply:
Sweet. Also, for reference, check out the code in CoreAbility. CoreAbility is the base of all abilities, so if you were wondering how it still works without the method, check `isEnabled` in there ;)

Reply:
Oooh I see. I looked at the wrong class. I'll check it out!